test/recipes/30-test_evp.t

```diff
@@ -124,6 +124,7 @@ my @defltfiles = qw(
                      evppkey_kdf_scrypt.txt
                      evppkey_kdf_tls1_prf.txt
                      evppkey_rsa.txt
+                     evppkey_rsa_sigalg.txt
```

Comment:
Once this is added to the fips provider this should be moved appropriately. However then RSA-SHA1 cannot be used as that should be used only for verification in FIPS.

Reply:
Done